File: WebServiceIdeias/src/main/java/br/com/gilmariosoftware/webserviceideias/modelo/Informacao.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package br.com.gilmariosoftware.webserviceideias.modelo;

/**
 *
 * @author gilmario
 */
public class Informacao {

    private String titulo;
    private String descricao;
    private TipoMensagem tipo;

    public Informacao() {
    }

    public Informacao(String descricao) {
        this.descricao = descricao;
        this.titulo = "Informação";
        this.tipo = TipoMensagem.INFORMACAO;
    }

    public Informacao(String titulo, String descricao) {
        this.titulo = titulo;
        this.descricao = descricao;
        this.tipo = TipoMensagem.SUCESSO;
    }

    public Informacao(String titulo, String descricao, TipoMensagem tipo) {
        this.titulo = titulo;
        this.descricao = descricao;
        this.tipo = tipo;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public TipoMensagem getTipo() {
        return tipo;
    }

    public void setTipo(TipoMensagem tipo) {
        this.tipo = tipo;
    }

    protected enum TipoMensagem {

        INFORMACAO, SUCESSO, ATANCAO, ERRO, FATAL
    }
}

File: WebServiceIdeias/src/main/webapp/resources/js/guardador/modelo/Informacao.js
Informacao = (function (titulo, descricao, tipo) {
    var titulo = titulo;
    var descricao = descricao;
    var tipo = tipo;

    this.getTitulo = (function () {
        return titulo;
    });
    this.getDescricao = (function () {
        return descricao;
    });

    this.getTipo = (function () {
        return tipo;
    });

    this.setTitulo = (function (titulo) {
        this.titulo = titulo;
    });

    this.setDescricao = (function (descricao) {
        this.descricao = descricao;
    });

    this.setTipo = (function (tipo) {
        this.tipo = tipo;
    });

});